examples/pytorch/question-answering/README.md

```diff
@@ -57,6 +57,28 @@ f1 = 88.52
 exact_match = 81.22
 ```
 
+### Fine-tuning T5 on SQuAD2.0
+
+This example code fine-tunes T5 on the SQuAD2.0 dataset.
```

Comment:
```suggestion
This example code fine-tunes T5 on the SQuAD2.0 dataset. It runs in ... min (with T5-small) or ... min (with T5-base) on a single tesla V100 16GB.
```
You could perhaps add similar figures to what is done for the BERT example. This is informative for people.